src/lib/features/export-import-toggles/import-validation-messages.ts

```diff
@@ -116,6 +118,14 @@ export class ImportValidationMessages {
             });
         }
 
+        if (dependencies.length > 0) {
+            errors.push({
+                message:
+                    'We detected the following dependencies in the import file that need to be created first:',
```

Comment:
we can remove "in the import file" since some people may not upload a file but paste content or use API